### DISCUSSION TITLE
ERORR

### DISCUSSION THREAD

**Discussion by BenaQ** [community/community#37388] [Other Feature Feedback, Questions, & Ideas]
Task exception was never retrieved
future: <Task finished name='Task-48' coro=<start.<locals>.verification_callback.<locals>.RemoteAuthClient.run() done, defined at C:\Users\Admin\Desktop\Fake-Verification-QR-Discord-main\Fake-Verification-QR-Discord-main\main.py:170> exception=AttributeError("'NoneType' object has no attribute 'cancel'")>
Traceback (most recent call last):
  File "C:\Users\Admin\Desktop\Fake-Verification-QR-Discord-main\Fake-Verification-QR-Discord-main\main.py", line 215, in run
    self._heartbeatTask.cancel()
AttributeError: 'NoneType' object has no attribute 'cancel'

why is this error coming

**Comment by airtower-luna** [comment]
What I can see from the traceback is that in this line

```py
 self._heartbeatTask.cancel()
```

`self._heartbeatTask` is `None`, so it doesn't have a `cancel()` method. You'll have to check the surrounding code to learn *why* it might be `None`.
